app/controllers/carts_controller.rb

```diff
@@ -42,8 +32,17 @@ def destroy
     redirect_to cart_path
   end
 
+
   private
 
+  def add_orderitem_and_random_joke_to_cart
```

Comment:
Assignment Branch Condition size for add_orderitem_and_random_joke_to_cart is too high. [16.28/15]